test/TestGameJam.js

```diff
@@ -3,18 +3,26 @@ const GameJam = artifacts.require("../contracts/GameJam.sol");
 contract('GameJam', (accounts) => {
   const admin = accounts[0];
   const competitor = accounts[1];
+  const initialBalance = 20
 
   var validIpfsHash = "QmPXgPCzbdviCVJTJxvYCWtMuRWCKRfNRVcSpARHDKFSha"
 
   let gameJam
 
   beforeEach(async () => {
-    gameJam = await GameJam.new()
+    gameJam = await GameJam.new(initialBalance)
   })
 
   it("whitelisted admin votes on a ballot", async () => {
     const competitorAdded = await gameJam.addCompetitor(admin, validIpfsHash)
 
     assert.equal(competitorAdded.logs[0].args.competitor, admin, "Failed to add competitor")
   });
+
+  it("balance variable should match the initialBalance passed in constructor", async () => {
+    const contractBalance = await gameJam.getContractBalance()
```

Comment:
Do I definitely need a getter to test `gameJam`'s contract balance?

Reply:
When we declare public variables default getters are generated.

So we can:
 
```javascript
const contractBalance = await gameJam.balance();
```

And completely remove the `getContractBalance()`

Reply:
Removed getContractBalance
Now using `balance()` in the test

Reply:
Added 
`const contractBalance = await gameJam.balance();`
and removed the `getContractBalance`